bin/sentry

```diff
@@ -6,8 +6,7 @@ error_reporting(E_ALL | E_STRICT);
 
 // TODO: if we could get rid of this and have composer figure things out it'd make it
 // a bit more sane
-require(__DIR__ . '/../lib/Raven/Autoloader.php');
-\Raven\Autoloader::register();
+require(__DIR__ . '/../vendor/autoload.php');
```

Comment:
I fixed this because I used this command to test whether the stacktrace was still sent correctly to the server, so I needed it to work correctly